src/app/components/footer/footer.component.html

```diff
@@ -1,36 +1,47 @@
 <footer class="pb-0 pt-2 bg-light">
   <div class="container">
-    <div class="row justify-content-between">
-      <div class="col-md-4">
-          <span placement="top" [ngbTooltip]="tipContent" i18n tabindex="0">Program versions</span>
-      </div>
-      <div class="col-md-5 pull-right">
-        <div class="row">
-          <div class="col-md-6">
+    <div class="col-md-12">
+      <nav class="row justify-content-between">
+        <ul class="row">
+          <li class="col-md">
+            <span placement="top" [ngbTooltip]="tipContent" i18n tabindex="0">Program versions</span>
+          </li>
+          <li class="col-md-auto">
             <i class="fa fa-envelope-o" aria-hidden="true"></i>
             <a class="text-dark" href="mailto:{{contactAddress}}">{{contactAddress}}</a>
-          </div>
-          <div class="col-md-4">
-            <i class="fa fa-github" aria-hidden="true"></i>
-            <a class="text-dark" href="https://github.com/zonemaster/zonemaster">Github</a>
-          </div>
+          </li>
+          <li class="col-md-auto">
+            <i class="fa fa-code" aria-hidden="true"></i>
+            <a class="text-dark" href="https://github.com/zonemaster/zonemaster">Source code</a>
+          </li>
+          <li class="col-md-auto">
+            <i class="fa fa-book" aria-hidden="true"></i>
+            <a class="text-dark" href="https://github.com/zonemaster/zonemaster#documentation">Documentation</a>
+          </li>
+        </ul>
+      </nav>
+
+      <div class="row">
+        <div class="col-md-auto">
+          <p class="copyright small" i18n>Copyright © AFNIC and The Swedish Internet Foundation. All Rights Reserved</p>
         </div>
       </div>
+
     </div>
 
-    <div class="row">
-      <div class="col-md-12">
-        <p class="copyright text-muted small" i18n>Copyright © AFNIC and The Swedish Internet Foundation. All Rights Reserved</p>
-      </div>
+    <div class="col-md-12" *ngIf="footerLogo !== ''">
+      <img src="{{footerLogo}}">
```

Comment:
There is no alt text for the image. Either provide one from the configuration (should probably be language specific with a default language) or assume it is only cosmetic (probably not) and put an empty alt.

Also I don't see any constraint on size, on any screen width. What is the recommended size for the logo?

Reply:
Alt parameter added. I do not think translation is required. I guess what makes most sense is to have the same string as in the logotype, and that is not translated.